### ISSUE TITLE
MaterialStackView sample error

### ISSUE THREAD

**Issue by sohayb** [CosmicMind/MaterialKit#96]
In the material stack view sample there's a problem in the code
````
materialStackView.distribution = .
````

**Comment by danieldahan** [comment]
I believe I deleted the MaterialStackView and opted to use MaterialLayout. It didn't make sense to write what StackView already provides. I am going to close this, but if you have any further issues, please re-open and let me know.

**Comment by sohayb** [comment]
Ah Okay, I kept it in the samples though

**Comment by danieldahan** [comment]
I believe I made the change a day or two ago and pushed the release. One of the aspects I am looking to improve on is a release and roadmap area. I appreciate your effort with the sample project. Thank you!

**Comment by sohayb** [comment]
Weird, because the sample now is based on the latest release examples. Sure thing, the least I can do :)

**Comment by danieldahan** [comment]
I delete the programmatic example for MaterialStackView. Can you confirm this?

**Comment by sohayb** [comment]
Nope, still there

**Comment by danieldahan** [comment]
in release 1.25.2?

**Comment by sohayb** [comment]
I cloned the master branch and it was there

**Comment by danieldahan** [comment]
We are talking about this class? MaterialStackView?

**Comment by sohayb** [comment]
````
MaterialKit -> Examples -> Programmatic -> StackView -> MaterialStackView -> ViewController
````
this is what I'm talking about.

**Comment by danieldahan** [comment]
I realized what happened. When I had deleted it from Xcode, it deleted the project but not the directory. I fixed that now and pushed a new version 1.25.3. Thank you again :)

**Comment by sohayb** [comment]
Sure thing :)
